packages/bundle/src/index-minimal.ts

```diff
@@ -4,6 +4,7 @@
 import { StrictStyleOptions, StyleOptions } from 'botframework-webchat-api';
 import * as decorator from 'botframework-webchat-api/decorator';
 import { Constants, createStore, createStoreWithDevTools, createStoreWithOptions } from 'botframework-webchat-core';
+import * as internal from 'botframework-webchat-component/internal';
```

Comment:
As we need this for Fluent, using dedicated bundle would mean UMD users need to include it in order for Fluent theme to work.